### ISSUE TITLE
Add check for non-unique logical paths

### ISSUE THREAD

**Issue by zimeon** [zimeon/ocfl-py#39]
See https://github.com/OCFL/fixtures/pull/65 which currently gets incorrectly reported as valid

**Comment by zimeon** [comment]
In E095_non_unique_logical_paths:
```
  "versions": {
    "v1": {
      "created": "2021-03-31T10:21:21.740101149-05:00",
      "state": {
        "07e41ccb166d21a5327d5a2ae1bb48192b8470e1357266c9d119c294cb1e95978569472c9de64fb6d93cbd4dd0aed0bf1e7c47fd1920de17b038a08a85eb4fa1": [
          "file-1.txt"
        ],
        "9fef2458ee1a9277925614272adfe60872f4c1bf02eecce7276166957d1ab30f65cf5c8065a294bf1b13e3c3589ba936a3b5db911572e30dfcb200ef71ad33d5": [
          "file-1.txt"
        ],
        "b3b26d26c9d8cfbb884b50e798f93ac6bef275a018547b1560af3e6d38f2723785731d3ca6338682fa7ac9acb506b3c594a125ce9d3d60cd14498304cc864cf2": [
          "file-3.txt", "file-4.txt", "file-3.txt"
        ]
      },
```

**Comment by zimeon** [comment]
```
(py38) simeon@RottenApple ocfl-py> ./ocfl-validate.py fixtures/1.0/bad-objects/E095_non_unique_logical_paths
[E095a] OCFL Object root inventory version v1 state has logical path file-1.txt used more than once. (see https://ocfl.io/1.0/spec/#E095)
[E095a] OCFL Object root inventory version v1 state has logical path file-3.txt used more than once. (see https://ocfl.io/1.0/spec/#E095)
INFO:ocfl.object:OCFL object at fixtures/1.0/bad-objects/E095_non_unique_logical_paths is INVALID
```

**Comment by zimeon** [comment]
Fixed in 1.2.2
